src/Tribe/RSVP.php

```diff
@@ -581,6 +661,184 @@ public function get_statuses_by_action( $action ) {
 		return $status_mgr->get_statuses_by_action( $action, 'rsvp' );
 	}
 
+	/**
+	 * Create an attendee for a RSVP ticket.
+	 *
+	 * @since TBD
+	 *
+	 * @param \Tribe__Tickets__Ticket_Object $ticket        Ticket object.
+	 * @param array                          $attendee_data Attendee data.
+	 *
+	 * @return int Attendee ID.
+	 *
+	 * @throws Exception
+	 */
+	public function create_attendee_for_ticket( $ticket, $attendee_data ) {
+		$rsvp_options = \Tribe__Tickets__Tickets_View::instance()->get_rsvp_options( null, false );
+
+		$required_details = [
+			'full_name',
+			'email',
+		];
+
+		foreach ( $required_details as $required_detail ) {
+			// Detail is not set.
+			if ( ! isset( $attendee_data[ $required_detail ] ) ) {
+				throw new Exception( sprintf( __( 'Attendee field "%s" is not set.', 'event-tickets' ), $required_detail ) );
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: ee8717204ecea04a3fa885279b4d9f66</sub>